fzf-tab.zsh

```diff
@@ -158,8 +185,9 @@ _fzf_tab_complete() {
     if [[ -n $choice ]]; then
         local -A v=("${(@0)${_fzf_tab_compcap[$choice]}}")
         local -a args=("${(@ps:\1:)v[args]}")
+        [[ -z $args[1] ]] && args=()  # don't pass an empty string
         IPREFIX=$v[IPREFIX] PREFIX=$v[PREFIX] SUFFIX=$v[SUFFIX] ISUFFIX=$v[ISUFFIX] \
-               builtin compadd "${args[@]:-Q}" -Q -- $v[word]
+               builtin compadd "${args[@]:--Q}" -Q -- $v[word]
```

Comment:
Is double `-Q` intentional?

Reply:
Yes. I should also fix it in master. It's expected to expand to `-Q` when `args` is empty, or `compadd` will treat it as an empty candidate.